config/zeus-sky.php

```diff
@@ -23,9 +23,14 @@
     'page_uri_prefix' => 'page',
 
     /**
-     * enable or disable FAQ addons.
+     * enable or disable individual Resources.
      */
-    'enableFaq' => true,
+    'enabled_resources' => [
+        LaraZeus\Sky\Filament\Resources\PostResource::class,
+        LaraZeus\Sky\Filament\Resources\PageResource::class,
+        LaraZeus\Sky\Filament\Resources\TagResource::class,
```

Comment:
we can't disable tags since it will break other resources

Reply:
Yes, i see. I will make 'tags' being always active and it is just being hidden in the Navigation Bar then, give me a second...